master/docs/manual/installation/installation.rst

```diff
@@ -75,7 +75,9 @@ If you cannot or do not wish to install the buildbot into a site-wide location l
 Running Buildbot's Tests (optional)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you wish, you can run the buildbot unit test suite.
+If you wish, you can run the Buildbot unit test suite. 
+You must not be using a python wheels packaged version of Buildbot or have specified the bdist_wheel flag when building. 
```

Comment:
I believe `bdist_wheel` is a command, not a flag.

I think an example of the command line with `bdist_wheel` would be great.